### PR TITLE
fix(pipeline): The context inside taskrun cannot be passed, resulting in the loss of some information during the execution of api-test executor

### PR DESCRIPTION
**Pull request by kakj-go**
#### What this PR does / why we need it:
The cluster information is lost during scenario execution, resulting in the failure of sending the request

#### Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=298355&issueFilter__urlQuery=eyJzdGF0ZXMiOls0NDEyLDQ1MzgsNDQxMyw0NDE0LDQ0MTUsNDQxNl0sImFzc2lnbmVlSURzIjpbIjEwMDA1NjAiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=1146&tab=BUG&type=BUG)

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |           fix The cluster information is lost during scenario execution, resulting in the failure of sending the request                  |
| 🇨🇳 中文    |        修复场景执行丢失集群信息导致发送请求失败      |

### CODE REVIEW

**Comment by kakj-go** [comment]
test image:

image

File: modules/pipeline/providers/reconciler/task_reconciler.go [hunk 1]
```diff
@@ -196,8 +198,11 @@ func (tr *defaultTaskReconciler) ReconcileNormalTask(ctx context.Context, p *spe
 		}
 
 		// generate framework to run task
-		framework := taskrun.New(ctx, task, executor, p, tr.bdl, tr.dbClient, tr.actionAgentSvc, tr.extMarketSvc, tr.clusterInfo)
+		framework = taskrun.New(ctx, task, executor, p, tr.bdl, tr.dbClient, tr.actionAgentSvc, tr.extMarketSvc, tr.clusterInfo)
+		return rutil.ContinueWorkingAbort
+	})
```
Comment (sfwn) [inline modules/pipeline/providers/reconciler/task_reconciler.go:203]: set default interval explicitly.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**